src/pymatgen/io/lobster/outputs.py

```diff
@@ -421,40 +434,39 @@ def __init__(
                 n_bonds = len(data_without_orbitals)
 
             labels: list[str] = []
-            atoms1: list[str] = []
-            atoms2: list[str] = []
+            atom1s: list[str] = []
```

Comment:
this rename looks accidental?

Reply:
No it's intended https://github.com/materialsproject/pymatgen/pull/3887#discussion_r1646898144